docs/README.md

```diff
@@ -28,3 +36,69 @@
 - [Writing performant React components](writing-performant-react-components.md)
 - [Deployment](deployment.md)
 - [Debugging](debugging.md)
+
+## Tools used
+
+- **pnpm**: a package manger
+  - pnpm's workspace feature is the foundation of the monorepo
+- **turborepo**: a monorepo scripting tool
+  - parallelize script execution
+  - manage execution dependency
+  - cache outputs to accelerate execution
+- **syncpack**: a monorepo dependency manager
+  - synchronize package dependencies
+  - validate dependency version ranges with configurable rules
+  - format and lint package json
+- **changeset**: a monorepo version manager
+  - increment semver in a topological way
+  - progressively compile release notes as PRs merge
+- **vite**: a bundler/dev server
+  - vite bundles monorepo apps for deployment
+  - vite's lib mode also builds and bundles some packages
+- **vitest**: a testing framework
+  - vitest is workspace-aware
+  - testing and bundling can use the same vite config
+  - vitest provides in-browser testing via playwright
+
+## The Basics
+
+### Local, compact chain state
+
+Minifront is a webapp blob which stores no state, and executes locally on the
+user's browser. Prax is a browser extension on the user's local machine, where
+it manages keys, configuration, and an abbreviated chain representing only the
+user's activity.
+
+[![finch's presentation at ZK8 describing the state commitment tree](https://img.youtube.com/vi/mHoe7lQMcxU/0.jpg)](https://www.youtube.com/watch?v=mHoe7lQMcxU)
+
+### Simplified architecture
```

Comment:
I think somewhere in one readme we should link to this web blog post we recently published: https://penumbra.zone/blog/faster-client-side-proving-with-parallelism